web.rb

```diff
@@ -166,15 +186,18 @@ def create_payment_intent(amount, source_id, payment_method_id, customer_id = ni
     :payment_method_types => ['card'],
     :description => "Example PaymentIntent",
     :shipping => shipping,
+    :return_url => return_url,
+    :confirm => return_url != nil,
```

Comment:
Is it always the case that if we have a `return_url` that we want to immediately confirm? They seem unrelated to me.

Reply:
removed